.circleci/config.yml

```diff
@@ -69,7 +77,7 @@ jobs:
   build:
     working_directory: ~/code
     docker:
-      - image: mbgl/android-ndk-r19:latest
+      - image: mbgl/android-ndk-r21e:latest
```

Comment:
I don't really understand what is the difference between 19 and 21. Can please somebody help me findout.

It was updated because I know for sure that r21e has the github cli

Reply:
You can find all available images and their contents here: https://github.com/mapbox/mbgl-ci-images/tree/master/images.

Reply:
thanks! but there is no version 19 🤔

Reply:
but it's present on [docker hub](https://hub.docker.com/layers/mbgl/android-ndk-r19/latest/images/sha256-a7f7f4db4c8f514db8b060c6d6bdc6e8ae6d50c3d0d9d1806648f7a0df49c443?context=explore)

Reply:
As I see the repo contains only `java-library` modules. Both the `r19` and the `r21e` has the same `openjdk-8-jdk-headless` installed. So I guess it's safe to change the image.
Is there anything I missed?

Reply:
r19 was renamed to r20 long time ago: https://github.com/mapbox/mbgl-ci-images/commit/5f0d151c881c1891608a5abfc196c12d1d77d058, so it is very outdated.